hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -529,8 +550,8 @@
             Verify the SETUP stage of enumeration.
 
             - Send a setup token with a valid setup packet containing the address(0), endpoint number(0)
```

Comment:
Do we have an opinion about the address and endpoint here? If not, maybe it makes sense to make this test more general?